tests/utils.py

```diff
@@ -33,7 +31,7 @@ def _build_patcher(self, target_module):  # pragma: no cover
 
     def __enter__(self):
         for patcher in self.patchers:
-            mocked_symbol = patcher.start()
+            patcher.start()
```

Comment:
I'd rather have: ``_mocked_symbol = patcher.start)``: since the call returns a value, let's tell the reader what that value is.

Reply:
Since the value is unused, we can show that to the reader by not assigning at all. It's the most explicit way to say that we don't care about the return value. I think it's easier to defer to the official [patch methods start and stop](https://docs.python.org/3/library/unittest.mock.html#patch-methods-start-and-stop) documentation, if we want to know the return value of `mock.patch.start()`.

Besides, if `patch.start()` was to be updated to return something else, chances are this code would not be updated, because the change does not actually impact the code, resulting in misleading documentation.

Finally, renaming `mocked_symbol` to `_mocked_symbol` does not remove the assign but never used warning. We could `# noqa: F841` this occurrence, but I think it would be best to remove the assignment.

Reply:
OK, let's remove that — we're never gonna use that return value anyway (and it's just test files, not the library core).